### ISSUE TITLE
Abandon service based theme

### ISSUE THREAD

**Issue by SameenAhnaf** [TeamNewPipe/NewPipe#9822]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Search function should be shown in a single line specifying what the app is able to search for. This is how what's new page should look if added as main page.

![Document 4_2](https://user-images.githubusercontent.com/79245641/219886280-65c017cc-5e9e-4340-b072-81930583d9cd.jpg)

In the middle of search bar and history, select any service with radio buttons.

![Document 3_1](https://user-images.githubusercontent.com/79245641/219886806-bbc9e951-6bf5-49e6-9507-bfd44617baff.jpg)

Short press on top menu of drawer should access about section. In the middle, subscriptions, playlists and other stuff will be placed. Kiosk pages will be placed at the bottom.

<img alt="Document_1" src="https://user-images.githubusercontent.com/79245641/219890476-35cefaeb-397d-4d31-9472-67d2f7fe08a3.jpg" width=340 />

### Why do you want this feature?

Newpipe follows a service based theme. It's counter-intuitive as subscriptions, history and playlists are shown as a unified UI. Implementing above suggestions will solve https://github.com/TeamNewPipe/NewPipe/issues/6272 and https://github.com/TeamNewPipe/NewPipe/issues/9632

**Comment by melihozkurt** [comment]
Good idea. I will try to adding in #9716 when designing search section. It can be mix of Firefox (simplicity  of choosing search engine) + Safari (showing suggestions + trends etc.)
